R/transformations.R

```diff
@@ -298,7 +298,7 @@ impute_simple <- function(object, value, na_limit = 0) {
 
   imp <- exprs(object)
   nas <- apply(imp, 1, prop_na)
-  imp <- imp[nas > na_limit, ]
+  imp <- imp[nas > na_limit,, drop = FALSE] #

```

Comment:
Just a quick style suggestion:

```suggestion
  imp <- imp[nas > na_limit, , drop = FALSE]
```